spec/features/category_edit_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'categories/edit', type: :view do
+  let(:user) { FactoryBot.create(:user) }
+  let(:category) { FactoryBot.create(:category) }
+
+  before(:each) do
+    login_as(user, scope: :user)
+    assign(:category, category)
+  end
+
+  it 'renders the edit category form' do
+    render
+
+    assert_select 'form[action=?][method=?]', category_path(category), 'post' do
+      assert_select 'input[name=?]', 'category[name]'
+
+      assert_select 'input[name=?]', 'category[icon]'
+    end
+  end
```

Comment:
- [ ] Good job making the required changes, Unfortunately, there are some tests still failing. Please fix them and make sure that all tests are passing.

![image](https://github.com/Salimer/Budget-app/assets/68796232/38807599-f499-423d-bd00-158917f43007)